src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -93,13 +145,19 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             // ActionMap should never point to nullptr
             FAIL_FAST_IF_NULL(cmd);
 
-            return !cmd.HasNestedCommands() && cmd.ActionAndArgs().Action() == ShortcutAction::Invalid ?
-                       nullptr : // explicitly unbound
-                       cmd;
+            return cmd;

```

Comment:
the left side disallows commands with nested commands . . . the right side doesn't. is that OK?

Reply:
oh, is it just because all commands with nested commands _are_ invalid, so we need to exclude them?

Reply:
The new `_ActionMap` does not store nested commands to begin with (those are only stored in `_NestedCommands`), so this is now an unnecessary check